streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetricsImpl.java

```diff
@@ -94,15 +97,16 @@ private String sensorName(String operationName, String entityName) {
         }
     }
 
-    private Map<String, String> tagMap(String... tags) {
+    public Map<String, String> tagMap(String... tags) {
```

Comment:
It uses the default tags from the `StreamsMetricsImpl` class so it is hard to make static.